packages/gp2-components/src/organisms/WorkingGroupCard.tsx

```diff
@@ -81,6 +83,7 @@ const WorkingGroupCard: React.FC<WorkingGroupCardProps> = ({
         <Subtitle noMargin accent="lead">
           {shortDescription}
         </Subtitle>
+        {tags.length > 0 && <TagList tags={tags.map((tag) => tag.name)} />}
```

Comment:
This is also limited to 3 tags
<img width="742" alt="Screenshot 2024-03-25 at 14 14 14" src="https://github.com/yldio/asap-hub/assets/16595804/d8bdaecd-d774-497e-af4d-06412776b2f6">